filament/include/filament/Options.h

```diff
@@ -438,6 +438,38 @@ struct SoftShadowOptions {
     float penumbraRatioScale = 1.0f;
 };
 
+/**
+ * View-level options for local IBLs.
+ */
+struct IblOptions {    
+    enum class IblTechnique : uint32_t {
+        IBL_INFINITE, IBL_FINITE_SPHERE, IBL_FINITE_BOX
```

Comment:
As implemented the mode "box" doesn't seem to work with `IndirectLight::setRotation`.

Reply:
I believe what you saw was the sphere's reciprocal radius overwriting the Y half extent of the box. 3adc8ec5279767b07c6389d192e69952a6d90b07 fixes this up.

Reply:
No I mean, that if we call setRotation() the IBL will be rotated but not the box (it's not a problem for the sphere).